<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>语雀备份助手</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon/logo</normaloff>:/icon/logo</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>语雀账户配置</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" rowspan="3">
          <widget class="QPushButton" name="btn_config">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>进行配置</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>（必填）USER_AGENT：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="cb_agent_state">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>未配置</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>（必填）ACCESS_TOKEN：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="cb_token_state">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>未配置</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>下载所有仓库</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btn_all_load">
            <property name="text">
             <string>加载所有仓库</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_all_download">
            <property name="text">
             <string>直接下载</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tv_all"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>下载指定仓库</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0">
          <item>
           <widget class="QLineEdit" name="edit_repo">
            <property name="placeholderText">
             <string>仓库路径，如：https://www.yuque.com/icheima/python</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_repo_load">
            <property name="text">
             <string>加载</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_repo_download">
            <property name="text">
             <string>直接下载</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="lv_repo"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>下载指定文档</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
          <item>
           <widget class="QLineEdit" name="edit_doc">
            <property name="placeholderText">
             <string>文档路径，如：https://www.yuque.com/icheima/python/env</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_doc_load">
            <property name="text">
             <string>加载</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_doc_download">
            <property name="text">
             <string>直接下载</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="text_doc"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>日志输出</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPlainTextEdit" name="edit_log">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
color: white;</string>
         </property>
         <property name="plainText">
          <string>&gt;&gt;</string>
         </property>
         <property name="backgroundVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
